modules/core/src/smithy4s/http/HttpBinding.scala

```diff
@@ -106,10 +106,14 @@ object HttpBinding extends ShapeTag.Companion[HttpBinding] {
   private[smithy4s] def fromHints(
       field: String,
       fieldHints: Hints,
-      inputOutput: Option[InputOutput]
-  ): Option[HttpBinding] = inputOutput.flatMap {
-    case InputOutput.Input  => fromHintsInput(field, fieldHints)
-    case InputOutput.Output => fromHintsOutput(field, fieldHints)
+      shapeHints: Hints
```

Comment:
@kubukoz I've changed this. Rendering an `Output` on the error type wouldn't quite work, because what's being serialised currently in our http integrations, is not so much the coproduct, but the individual member. 

Instead, we're relying on the presence of the `Error` hint